tests/local/sql/test_missing_and_invalid_metric_configurations.py

```diff
@@ -19,21 +19,18 @@ class TestMissingAndInvalidMetricConfigurations(SqlTestCase):
 
     def setUp(self) -> None:
         super().setUp()
-        self.sql_create_table(
-            self.table_name,
-            ["id VARCHAR(255)",
-             "name VARCHAR(255)",
-             "size INTEGER"],
+        self.sql_create_test_table(
+            [self.warehouse.dialect.declare_string_column_sql("id"),
```

Comment:
@marciogualtieri Can you create convenience methods in SqlTestCase so that this looks like `self.declare_varchar_column('id')` and apply that to all other create test table invocations as well?
tnx

Reply:
I couldn't wait for you to review the BigQuery changes I made available Saturday, thus, I already have Athena changes in place. I could have created another branch, but this would cause me to have to replicate many change in different branches. I've already created convenience methods that have not been committed. Feel free to keep reviewing though, as I haven't done more than small refactoring to BigQuery. :+1:

Reply:
Ok. So if I get it correctly, my requested changes are in another branch.  So in that case go ahead and merge this branch.  And the requested change will come later no prob.

Reply:
No, it's the opposite. I said that I'm using this branch for both BigQuery and Athena. I have now committed all changes to this branch. The change you have requested was done before you asked for them. Your browser might be caching and old page. The change was already in place when you wrote your comment 12 hours ago.